src/printf/printf.c

```diff
@@ -261,11 +260,11 @@ static inline void out_wrapped_function(char character, void* wrapped_function,
 
 // internal secure strlen
 // @return The length of the string (excluding the terminating 0) limited by 'maxsize'
-static inline unsigned int strnlen_s_(const char* str, size_t maxsize)
+static inline printf_uint_t strnlen_s_(const char* str, size_t maxsize)
```

Comment:
Maybe I could just make it size_t?

Reply:
Good suggestion, done in an additional commit.